tests/unit/domain/test_chat_context.py

```
@@ -0,0 +1,454 @@
+import pytest
+from uuid import uuid4
+from datetime import datetime, timedelta
```

Ruff (0.11.9)

`datetime.timedelta` imported but unused

Remove unused import: `datetime.timedelta`

(F401)